app/controllers/Contributions.java

```diff
@@ -92,7 +92,7 @@
     public static final Form<ThemeListTransfer> THEMES_FORM = form(ThemeListTransfer.class);
     public static final Form<User> AUTHORS_FORM = form(User.class);
 
-	private static BufferedReader br;
+    private static BufferedReader br;
```

Comment:
we need to come up with ways for avoiding these type of changes, that are not really changes, but merely replacement of  spaces for tabs or viceversa @MarceMMAD @rebearteta @javierpf @jorgeramirez any idea is welcomed